vendor.mod

```diff
@@ -6,6 +6,9 @@ module github.com/docker/cli
 
 go 1.22.0
 
+// FIXME(thaJeztah): testing https://github.com/moby/moby/pull/49005
+replace github.com/docker/docker => github.com/thaJeztah/docker v24.0.0-rc.1.0.20241207110217-3014d6d7a324+incompatible
```

Comment:
Temporary commit to test with the mobychanges from  https://github.com/moby/moby/pull/49005 (all looks good, so I'll remove it)